romancal/associations/asn_from_list.py

```diff
@@ -43,7 +43,7 @@ def asn_from_list(items, rule=DMS_ELPP_Base, **kwargs):
     return asn
 
 
-class Main:
+def _cli(args=None):
```

Comment:
Including a class as a script in `pyproject.toml` as is done on main:
https://github.com/spacetelescope/romancal/blob/bee71bad93ef6c0252cdcc86933289061ed8f69d/pyproject.toml#L95
results in an installed script containing the following line:
```python
sys.exit(Main())
```
This is problematic because `Main()` is "truthy" resulting in the script returning 1 on success. By switching this to a function (with no return value) the script becomes:
```
sys.exit(main())
```
and the no return value (None) is "falsey" leading to a return code of 0 (success) when the script succeeds.